### ISSUE TITLE
Team: Echoes

### ISSUE THREAD

**Issue by jntrnr** [langjam/jam0001#267]
Add your votes to this issue by using emoji reactions

**Comment by pitust** [comment]
+1 since it doesn't include comment syntax :^)

**Comment by ZikiFlicky** [comment]
@pitust Yeah, that... I didn't have enough time to implement what I wanted (which was pretty close to what JT implemented in [his last video](https://www.youtube.com/watch?v=CccI8vqsO8k)), but I think the Readme section was a good enough non-code take on the theme 🤷‍♂️. Overall, the language is still really broken, so I will probably keep working on it at some point in the near future, and hopefully do stuff properly this time.
